packages/firestore/src/protos/firestore_proto_api.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Do you still need this file?

Reply:
Yup, it breaks the TypeScript compile even without the API Extractor.